pkg/inject/env.go

```diff
@@ -0,0 +1,6 @@
+package inject
+
+const (
```

Comment:
There are some more of these env const keys used in jaeger injector which can be moved here. I will do this as a separate PR.